File: src/test/java/io/github/marmer/ApplicationTest.java
package io.github.marmer;

import lombok.SneakyThrows;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

class ApplicationTest {

    @Test
    @DisplayName("Hello Test")
    @SneakyThrows
    void someStaticJavaMethod_someStaticJavaMethod() {
        ApplicationKt.main(new String[]{"Hundekuchen", "Mettbrötchen"});
        HelloJava.someStaticJavaMethod("foo", "bar");
    }

}

File: src/test/kotlin/io/github/marmer/ApplicationTest.kt
package io.github.marmer

import org.junit.jupiter.api.Test

class ApplicationTest {

    @Test
    fun helloTest() {
        main(arrayOf("Hundekuchen", "Mettbrötchen"))
        HelloJava.someStaticJavaMethod("foo", "bar")
    }
}
